app/assets/javascripts/smoothScroll.js

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
On Firefox + Linux the smooth scroll is not happening and the positioning after the scroll is not well aligned.

Reply:
Nevermind, I think it was still loading. It worked fine ;)